pandas/core/format.py

```diff
@@ -23,6 +24,9 @@
 import itertools
 import csv
 
+class FormatWarning(Warning):
+    pass
```

Comment:
I did it from @jreback suggestion on July 15th https://github.com/pydata/pandas/pull/10570#issuecomment-121603671 But I don't have any strong feelings. Let me know what you want this to be.